rpm/openssl-pkcs11-sign-provider.spec

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2023 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+Name:          openssl-pkcs11-sign-provider
+Version:       1.0.0
+Release:       %autorelease
+Summary:       A PKCS#11 provider for OpenSSL 3.0 (private key operations only)
+License:       Apache-2.0
+URL:           https://github.com/opencryptoki/%{name}
+Source:        %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+
+Requires:      openssl >= 3.0.8
+
+BuildRequires: openssl-devel >= 3.0.8
+BuildRequires: opencryptoki-devel >= 3.17.0
+BuildRequires: gcc
+BuildRequires: g++
+BuildRequires: autoconf-archive
+BuildRequires: automake
+BuildRequires: libtool
+# for test
+BuildRequires: openssl >= 3.0.8
+BuildRequires: gnutls-utils
+BuildRequires: sed
+BuildRequires: opencryptoki >= 3.17.0
+
+%description
+This package contains a provider module for OpenSSL 3.0, interfacing to
+PKCS#11 for operations with private keys in PKCS#11 tokens.
+
+%global modulesdir %(pkg-config --variable=modulesdir libcrypto)
+
+%prep
+%autosetup
+
+%build
+%configure --libdir=%{modulesdir}
+%make_build
+
+%install
+%make_install
+
+%files
+%license COPYING
+%doc README openssl-*.cnf.sample
```

Comment:
Add CHANGES.md to %doc?

Reply:
CHANGES.md is currently not part of the distribution tarball. So it can not be included for v1.0.0. 

My personal opinion is, that `CHANGES.md` should not be part of the package for the following reasons:

- this specfile is meant for packaging the (unmodified) upstream releases. It maybe different from the one a package maintainer for a distribution will write.
- the changelog of the upstream releases can be looked up in github (via `URL`). There is no need to duplicate the information and include it in the RPM.
- the RPM has its own changelog. Maybe a package maintainer will use the information in `CHANGES.md` and copy it the the `%changelog` section.

But I'm open for arguments to include the `CHNAGES.md` in future versions/distribution-tarballs.